concordium-node/src/bin/cli.rs

```diff
@@ -138,10 +140,23 @@ async fn main() -> anyhow::Result<()> {
         None
     };
 
-    if let Some(ref import_path) = conf.cli.baker.import_path {
-        info!("Starting out of band catch-up");
-        consensus.import_blocks(import_path.as_bytes());
-        info!("Completed out of band catch-up");
+    // Out-of-band catch-up
+    if let Some(ref import_blocks_from) = conf.cli.baker.import_blocks_from {
+        let from: anyhow::Result<String> =
+            download_catchup_files(import_blocks_from, data_dir_path).await;
+        match from {
+            Ok(from) => {
+                info!("Starting out of band catch-up");
+                consensus.import_blocks(from.as_bytes());
+                info!("Completed out of band catch-up");
+                if let Err(e) = std::fs::remove_file(from) {
```

Comment:
We only want to delete the file when it was provided as a URL and downloaded by the node.